tests/replication/slave/storage/test_events.py

```diff
@@ -26,7 +28,9 @@
 
 
 def dict_equals(self, other):
-    return self.__dict__ == other.__dict__
+    me = encode_canonical_json(self._event_dict)
```

Comment:
this is going to fail on debian unless we backport frozendict 1.x... which we probably should, tbh, but is this really the best way to compare a pair of frozendicts?